test/integration/instance.test.js

```diff
@@ -981,6 +981,20 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
       });
     });
 
+    it('does not update timestamps when nothing changed', function() {
+      var User = this.User;
+      return User.create({ username: 'John' }).then(function() {
+        return User.findOne({ username: 'John' }).then(function(user) {
+          var updatedAt = user.updatedAt;
+          return this.sequelize.Promise.delay(2000).then(function() {
```

Comment:
Please use fake timers http://sinonjs.org/docs/#clock to artifically advance time